specification/visualstudio/resource-manager/Microsoft.VisualStudio/preview/2018-08-01-preview/Projects.json

```diff
@@ -0,0 +1,717 @@
+        "operationId": "Projects_Update",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The request data.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ProjectResourceUpdateParameters"
+            }
+          },
+          {
+            "in": "path",
+            "name": "rootResourceName",
+            "description": "Name of the Team Services account.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "resourceName",
+            "description": "Name of the Team Services project.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation succeeded. The response contains the details of the updated team project resource.",
+            "schema": {
+              "$ref": "#/definitions/ProjectResource"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Update a project resource": {
+            "$ref": "./examples/UpdateProjectResource.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PipelineBootstrapConfiguration": {
+      "description": "Defines bootstrap configuration for a CI/CD pipeline.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the bootstrap configuration.",
+          "type": "string"
+        },
+        "templateId": {
+          "description": "Identifier of the pipeline template used for configuring pipeline.",
+          "type": "string"
+        },
+        "templateParameters": {
+          "description": "Dictionary of inputs used for configuring pipeline. The keys for inputs are defined by pipeline template.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "repository": {
+          "description": "Code repository where the application source resides.",
+          "$ref": "#/definitions/CodeRepository"
+        }
+      },
+      "required": [
+        "name",
+        "templateId"
+      ]
+    },
+    "Resource": {
+      "description": "A generic Azure Resource Manager resource.",
+      "x-ms-azure-resource": true,
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Unique identifier of the resource.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "location": {
+          "description": "Resource location.",
```

Comment:
`"location"` isn't readonly here, which doesn't match the pattern of `"type"` and `"name"`. Was this deliberate?

Reply:
done